docs/resources/virtual_environment_metrics_server.md

```diff
@@ -27,6 +27,15 @@ resource "proxmox_virtual_environment_metrics_server" "graphite_server" {
   port   = 2003
   type   = "graphite"
 }
+
+resource "proxmox_virtual_environment_metrics_server" "opentelemetry_server" {
+  name                 = "example_opentelemetry_server"
+  server               = "192.168.5.2"
+  port                 = 4318
+  type                 = "opentelemetry"
+  opentelemetry_proto  = "https"
+  opentelemetry_path   = "/v1/metrics"
+}
```

Comment:
the docs for FWK-based resources of the provider are generated from the code, so you'd need to add this section to [resource.tf](https://github.com/infra-foundery/terraform-provider-proxmox/blob/f8e96fd79eb3c3f72e12b0b06cf32f68aeeb66b7/examples/resources/proxmox_virtual_environment_metrics_server/resource.tf) and then run `make docs` to generate the md files.